dashboard/app/models/levels/studio.rb

```diff
@@ -56,6 +56,8 @@ class Studio < Grid
     delay_completion
     floating_score
     goal_override
+    tap_svg_to_run_and_reset
```

Comment:
I find the name a bit confusing because it refers to a technology that we use in a variety of places rather than a component or UI element.  Not a big deal, but I'd probably call it something more like `tap_playarea_to_run_and_reset` or whatever else we want to call this part of the UI.